imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -296,14 +297,19 @@ def couple(self) -> None:
                     modribmsw_arrays["rib_sprinkling_demand"] = self.ribasim_user_demand
                     n_users = np.shape(self.mapping.msw2rib["sw_sprinkling"])[0]
                     n_priorities = np.size(self.ribasim_user_demand) // n_users
-                    self.ribasim_user_demand.resize(n_priorities, n_users)
-                    self.ribasim_user_demand0 = self.ribasim_user_demand.copy().reshape(
-                        n_priorities, n_users
-                    )
-                    self.ribasim_user_demand0[self.ribasim_user_demand0 > 0] = 1.0
-                    too_many = (
-                        np.where(self.ribasim_user_demand0.sum(axis=0) > 1)[0] + 1
+                    self.ribasim_user_demand.resize(n_users,n_priorities)
```

Comment:
Perhaps add a comment on why this mask is more complicated than for the other 'tradiational' arrays?